src/Hintman/Hint.hs

```diff
@@ -45,15 +46,25 @@ getModifiedFiles
     :: forall m env . (MonadIO m, WithLog env m)
     => PrInfo
     -> m [ModifiedFile]
-getModifiedFiles prInfo@PrInfo{..} = fmap catMaybes $ traverse toModifiedFile $ filter ((/=) Deleted . fileDeltaStatus) prInfoDelta
+getModifiedFiles prInfo@PrInfo{..} = fmap catMaybes $ traverse toModifiedFile $ filesForReview prInfoDelta
   where
+    -- | Sifts diffs, removing binary files and files that were removed.
+    filesForReview :: [FileDelta] -> [FileDelta]
+    filesForReview = filter notBinaryOrRemoved
+
+    notBinaryOrRemoved :: FileDelta -> Bool
+    notBinaryOrRemoved file = (fileDeltaStatus file /= Deleted) && (fileDeltaContent file /= Binary)
```

Comment:
This function name is a bit confusing, it should be "not deleted" instead. But generally, I think that you don't need this function, just move this predicate into `filesForReview` directly, you've already wrote that in the documentation to that function :+1: